tests/unit/btree_stress_test.c

```diff
@@ -45,13 +44,12 @@ insert_thread(void *arg);
 static void
 insert_tests(cache           *cc,
              btree_config    *cfg,
-             platform_heap_id heap_id,
+             platform_heap_id hid,
              btree_scratch   *scratch,
              mini_allocator  *mini,
              uint64           root_addr,
              int              start,
-             int              end,
-             platform_heap_id hid);
```

Comment:
Strangely, the same bug crept into this interface, too. Delete this arg, retain the `hid` arg on L47 above.